io/disk/fiotest.py

```diff
@@ -44,21 +45,46 @@ def setUp(self):
         """
         Build 'fio'.
         """
-        tarball = self.fetch_asset('http://brick.kernel.dk/snaps/fio-2.1.10.tar.gz')
+        default_url = "http://brick.kernel.dk/snaps/fio-2.1.10.tar.gz"
+        self.url = self.params.get('fio_tool_url', default=default_url)
```

Comment:
i think global assignment is not required for `self.url` as outside of `setUp` routine it not used